typescript/db/src/repository/index.ts

```diff
@@ -134,7 +134,10 @@ export const repository: Repository = {
             await getPrismaClient()
           ).label.update({
             where,
-            data: castObjectNullsToUndefined(label),
+            data: {
+              ...castObjectNullsToUndefined(label),
+              labelClassId: label.labelClassId,
```

Comment:
I had to add this because otherwise we could not set a label to the `None` value in the interface. `None` is not a label class, it just means that the label class is `null`. But since we were casting all the `null` to `undefined` whenever we had a `{labelClass:null}` input to this function it would never be applied

Reply:
OK so this is what I was thinking :) Thanks for the clarification!
So now with your changes, we respect the fact that `labelClassId` is optional in the `Label` model of our Prisma schema. Perfect 👍